nixos/modules/services/networking/wireguard.nix

```diff
@@ -148,6 +148,15 @@ let
         for matching all IPv6 addresses.'';
       };
 
+      allowedIPsAsRoutes = mkOption {
+        example = false;
```

Comment:
Why is example=false? (I'm not sure precisely what this controls.)

Reply:
It will be displayed as part of our generated documentation: `https://nixos.org/nixos/options.html#zsh.enable`
In case of boolean values this is kind of obvious, which is why it is often left out.

Reply:
Makes sense!